recipe/meta.yaml

```
@@ -8,10 +8,10 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sqlglot-{{ version }}.tar.gz
-  sha256: a64c99d0f512cff05c742b42b2f115327a56d4552ddde821b9158e94da782e36
+  sha256: 131d47163696228201224504e5a8c9f189134e9acece4c6336bb8534bd08b906
 
 build:
-  skip: true  # [py<37]
+  skip: true  # [py<38]
```

```suggestion
  skip: true  # [py<37]
```
Upstream still declares compatibility with 3.7
https://github.com/tobymao/sqlglot/blob/v25.10.0/setup.py#L29